.github/workflows/web-scan.yml

```diff
@@ -0,0 +1,48 @@
+name: 'Security web scan for zot'
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+  release:
+    types:
+      - published
+
+permissions:
+  contents: read
+  issues: write
+
+jobs:
+  zap_scan:
+    runs-on: ubuntu-latest
+    name: Scan the webapplication
+    strategy:
+      matrix:
+        flavor: [zot-minimal-linux-amd64, zot-linux-amd64]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          ref: main
+      - name: Setup
+        run: |
+          # upload images, zot can serve OCI image layouts directly like so
+          mkdir /tmp/zot
+          skopeo copy --format=oci docker://busybox:latest oci:/tmp/zot/busybox:latest
+          # start zot
+          docker pull ghcr.io/project-zot/zot-minimal-linux-amd64:latest
```

Comment:
Is there a guarantee this is the image produced from this commit? It doesn't seem like we are waiting for the other action building the image to finish. Maybe this logic should be in that workflow?

Also the flavor variable is not used here, but is used in docker run, below